AtlasLootClassic_Collections/data.lua

```diff
@@ -1412,6 +1474,57 @@ data["MidsummerFestival"] = {
 	},
 }
 
+data["Brewfest"] = {
+	name = AL["Brewfest"],
+	ContentType = WORLD_EVENT_CONTENT,
+	LoadDifficulty = NORMAL_DIFF,
+	TableType = NORMAL_ITTYPE,
+	gameVersion = 2,
+	items = {
+		{ -- Brewfest
+			name = AL["Brewfest"],
+			[NORMAL_DIFF] = {
+				{ 1,  34008 }, -- Blix's Eyesight Enhancing Romance Goggles
+				{ 2,  33968 }, -- Blue Brewfest Hat
+				{ 3,  33864 }, -- Brown Brewfest Hat
+				{ 4,  33967 }, -- Green Brewfest Hat
+				{ 5,  33969 }, -- Purple Brewfest Hat
+				{ 6,  33863 }, -- Brewfest Dress
+				{ 7,  33862 }, -- Brewfest Regalia
+				{ 8,  33966 }, -- Brewfest Slippers
+				{ 9,  33868 }, -- Brewfest Boots
+				{ 11,  33016 }, -- Blue Brewfest Stein
```

Comment:
The filled Stein's (33017, 33018, 33019, 33020) are the same item as the Blue Brewfest Stein (33016).
You get the Blue Brewfest Stein from a quest and then you can fill it on one of 4 different Festive Kegs.

Reply:
I have included the Filled Steins cause each of them has different quote.

Reply:
Yeah i mean i get that but imo it would be enough to just show `33016` bcs the other ones are not obtainable through the brewfest vendors/quests.
But thats was just my opinion. If you think this is better, im ok with that 👍